src/support_function.jl

```diff
@@ -1,25 +1,25 @@
-# =======================================
-# Support function of an abstract LazySet
-# =======================================
+# =============================
+# Support function of a LazySet
+# =============================
 
 """
-    ρ(d::Vector{Float64}, sf::LazySet)::Float64
+    ρ(d::Vector{N}, S::LazySet)::N where {N<:Real}
```

Comment:
can you let the direction be an `AbstractVector{N}`?

Reply:
Sure, good point.